test/suites/base.bash

```diff
@@ -285,7 +285,7 @@ fi
     TEST "Too new source file"
 
     touch new.c
-    touch -t 203801010000 new.c
+    touch -d "1 year" new.c
```

Comment:
As indicated by the failing tests, we can't assume that `touch` is of the GNU kind.

Reply:
Good point. I updated it to use the January 3rd of the following year, so it can be 2-367 days in the future.